checksit/cvs.py

```diff
@@ -25,26 +25,53 @@ def _load(self, vocab_id):
         vocab_file = os.path.join(vocabs_dir, f"{vocab_id}.json")
         self._vocabs[vocab_id] = json.load(open(vocab_file))
 
+    def _load_from_url_ncas(self, vocab_id_url):
+        vocab_id_url_base = vocab_id_url.split("/__latest__")[0]
+        vocab_id_url_base = vocab_id_url_base.replace(
+            "raw.githubusercontent.com", "github.com"
+        )
+        latest_version = requests.get(
+            f"{vocab_id_url_base}/releases/latest"
+        ).url.split("/")[-1]
+        vocab_id_url = vocab_id_url.replace("__latest__", latest_version)
+        res = requests.get(vocab_id_url.replace("__URL__", "https://"))
+        if res.status_code == 200:
+            vocab_list = res.json()
+        else:
+            print(f"[WARNING] Failed to load vocab: {vocab_id_url}")
+        return vocab_list
+
+    def _load_from_url_esacci(self, vocab_id_url):
+        res = requests.get(vocab_id_url)
+        if res.status_code == 200:
```

Comment:
Quick note on best practice, you can look to minimise indentation (i.e multiple nested ifs) by dismissing some cases immediately. In this case, you could switch to the following:
```
if res.status_code != 200:
    print('Warning...')
    return []
# Now the code can continue without 'else' because the only way to get to this point is if the status code is ok
js = res.json()
```

Also another note, this function returns the vocab list, but vocab list may not be defined at the end if specific things are not true. Best practice is to set `vocab_list = []` at the start of the function